### DISCUSSION TITLE
Metaverse.dev product offering and structure

### DISCUSSION THREAD

**Discussion by mrmetaverse** [AngellXR/metaversedev#11] [Ideas]
Curious, there are a number of directions the metaverse.dev website could go. Helping people achieve their goals of becoming metaverse creators is a noble goal that could take numerous routes. 

Do we take a blog approach like dev.to? 
[![image](https://user-images.githubusercontent.com/63426722/160496269-f9eae42d-9e80-453f-aa9f-eb00e4d9090e.png)](https://dev.to/)
https://dev.to/

Do we take a open video approach like youtube? 
[![image](https://user-images.githubusercontent.com/63426722/160496347-a3d3eb1f-7427-4678-b0d3-c1840d5bf098.png)](https://www.youtube.com/)
https://www.youtube.com/

Do we structure content into video series like udemy? 
[![image](https://user-images.githubusercontent.com/63426722/160496458-777a0065-8a59-4b34-bdde-60999433fa86.png)](https://www.udemy.com/)
https://www.udemy.com/

Do we go ultra-lightweight with a checklist and URLs? 
![image](https://user-images.githubusercontent.com/63426722/160496541-77a50ac1-1305-4c45-8db2-4b424933aea4.png)

Do we provide peer-tutoring options? 1:1 matching and mentoring? 
![image](https://user-images.githubusercontent.com/63426722/160496611-7a8c3848-071b-4deb-85d6-ab6263dc27aa.png)

Do we make an open lesson plan like the odin project? 
[![image](https://user-images.githubusercontent.com/63426722/160496669-c6c8729e-9f4a-4492-b4b5-740e060c4902.png)](https://www.theodinproject.com/)
https://www.theodinproject.com/

**Comment by msub2** [comment]
I'm partial to open lesson plans, if only because I got my start coding through things like Codecademy and freeCodeCamp. Combining open lesson plans with one or more avenues for students to collaborate/ask questions (forums, servers, etc) is what I'd personally find most appealing.

**Reply by mrmetaverse** [reply]
definitely agree. I have enjoyed Bruno Simons course for how in-depth it is, while remaining self paced.
